src/Microsoft.Azure.CosmosRepository/Extensions/ExpressionExtensions.cs

```diff
@@ -37,23 +37,44 @@ internal static class ExpressionExtensions
         this Expression<Func<T, bool>> first,
         Expression<Func<T, bool>> second) => first.Compose(second, Expression.Or);
 
-    internal static PropertyInfo GetPropertyInfo<TSource, TProperty>(this Expression<Func<TSource, TProperty>> propertyLambda)
```

Comment:
Same thing here, we'll need an overload, one that simply delegates to the other.

Reply:
I have split the code slightly to get the same effect as the original GetPropertyInfo had.